inst/include/frogger.hpp

```diff
@@ -30,7 +30,51 @@ struct Leapfrog {
     State initial_state = {};
     initial_state.reset();
     if constexpr (ModelVariant::run_demographic_projection) {
-      initial_state.dp.p_total_pop = pars.dp.base_pop;
+
+      // Initialise base year ouputs
+      const auto& p_dp = pars.dp;
+      auto& is_dp = initial_state.dp;
+      int t0 = 0;
+      
+      is_dp.p_total_pop = p_dp.base_pop;
+
+      // Initialise births and deaths in base year. This involves some
+      // approximations because the model does not conduct a full
+      // cohort component projection for this step.
+
+      // Births in base year
+      is_dp.births = 0.0;
+      for (int af = 0; af < SS::p_fertility_age_groups; ++af) {
+	
+	// female population exposed to births: average of current age a population
+	// and current age a+1 population survived backwards one year = average of current
+	// and previous year population. (Does not account for any base year net migration)
+
+	const auto a = SS::p_idx_fertility_first + af;
+	auto female_fertility_population_a = is_dp.p_total_pop(a, SS::FEMALE) + is_dp.p_total_pop(a+1, SS::FEMALE) / p_dp.survival_probability(a+1, SS::FEMALE, t0);
+	is_dp.births += female_fertility_population_a * 0.5 * p_dp.age_specific_fertility_rate(af, t0);
+      }
+
+      // Deaths in base year
+      // Note: this calculation would **probably** be more accurate if it survived the
+      // population 1 year backwards before calculating deaths (discussed with Rob
+      // Glaubius 7 June 2025)
+      for (int g = 0; g < SS::NS; ++g) {
+	      
+	// (a) age 0 deaths
```

Comment:
Whats the purpose of the A/B/C notation?

Reply:
Just to denote three cases or steps; can delete it if it does not help clarity. Or change it to `// Case 1: ...`, `// Case 2: ...`, etc.